torchvision/prototype/transforms/_geometry.py

```diff
@@ -512,21 +503,23 @@ def _get_params(self, flat_inputs: List[Any]) -> Dict[str, Any]:
 
         half_height = height // 2
         half_width = width // 2
+        bound_height = int(distortion_scale * half_height) + 1
+        bound_width = int(distortion_scale * half_width) + 1
```

Comment:
Cleaning up the code a bit to avoid repeated estimations and unnecessary castings.